records_versioning/services.py

```
@@ -0,0 +1,255 @@
+
+IGNORE_FIELDS = {"id", "pk", "created_at", "updated_at"}
+
+RESOURCE_MAP = {
+    'Patient': ('patients_core', 'Patient'),
+    'Encounter': ('diab_encounters', 'Encounter'),
+    'LabResult': ('diab_labs', 'LabResult'),
+    'MedicationOrder': ('diab_medications', 'MedicationOrder'),
+}
+
+
+def _compute_snapshot(instance: Any) -> Dict[str, Any]:
```

Ruff (0.12.2)

Dynamically typed expressions (typing.Any) are disallowed in `instance`

(ANN401)